dm/ctl/master/query_status.go

```diff
@@ -24,6 +24,20 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const stageError = "Error"
+
+type taskResult struct {
+	Result bool        `json:"result"`
+	Msg    string      `json:"msg"`
+	Tasks  []*taskInfo `json:"tasks"`
+}
+
+type taskInfo struct {
+	TaskName   string   `json:"taskName,omitempty"`
+	TaskStatus string   `json:"taskStatus,omitempty"`
```

Comment:
how about use same format with others, changed to `task-status`? and also the task-name.

Reply:
tag generated by protobuf is `CamelCase` :sweat_smile:

Reply:
maybe we should keep same with tag in other commands' output?